src/components/shared/__tests__/DocumentTitle.js

```diff
@@ -0,0 +1,61 @@
+import DocumentTitle from 'shared/DocumentTitle';
+import React from 'react';
+import { getComponentWithTheme } from 'testUtils/renderUtils';
+import { render } from '@testing-library/react';
+
+let initialTitle = '';
+
+const getComponent = props => {
+  const propsWithChildren = Object.assign({}, props, { children: <p /> });
+
+  return getComponentWithTheme(DocumentTitle, propsWithChildren);
+};
+const renderWithProps = props => render(getComponent(props));
+
+describe('DocumentTitle', () => {
+  beforeAll(() => {
+    initialTitle = document.title;
+  });
+
+  afterAll(() => {
+    document.title = initialTitle;
+  });
+
+  beforeEach(() => {
+    document.title = initialTitle;
+  });
+
+  it('renders without crashing', () => {
+    renderWithProps({});
+  });
+
+  it(`sets title to 'Giant Swarm' if no title prop is provided`, () => {
+    renderWithProps({});
+
+    expect(document.title).toBe('Giant Swarm');
```

Comment:
Shouldn't we be using a constant? I've seen `'Giant Swarm'` at least three times spreaded across the codebase